docs/querying/basics.md

```diff
@@ -224,6 +234,15 @@ Here are some examples of valid time durations:
     5m
     10s
 
+
+As of version 2.52, durations can also be represented by float values, implying the number of seconds. This is an experimental feature and might still change.
```

Comment:
With the pedantic terminology usage, as discussed above, this should read like this:

"As of version 2.53, time durations can also be represented using the syntax of [float literals](#float-literals), implying the number of seconds of the time duration. This is an experimental feature and might still change."